src/App.tsx

```diff
@@ -183,10 +208,12 @@ function APP() {
     { label: "HF", options: opts_hf },
     { label: "RHF", options: opts_rhf },
   ];
-  let opts_year = [2019, 2018, 2017, 2016];
 
-  const input_data = {
-    selected_unit: treatment_units,
+  const input_data: {
+    selected_unit: string[];
+    selected_year: number;
+  } = {
+    selected_unit: treatment_units as string[],
     selected_year: selected_year,
   };
```

Comment:
You shouldn't need to specify types here?

```suggestion
  const input_data = {
    selected_unit: treatment_units,
    selected_year: selected_year,
  };
```

Is TS not able to infer the correct type like this?

Reply:
ah nice! it didn't at first, but it does now - there was som initial inference problems..